packages/sdk/docker-bake.hcl

```diff
@@ -5,22 +5,27 @@ target "default" {
   inherits = ["docker-metadata-action", "docker-platforms"]
   args = {
     ALTO_VERSION                      = "0.0.4"
-    BASE_IMAGE                        = "debian:bookworm-20250224"
+    CARTESI_DEVNET_VERSION            = "2.0.0-alpha.3"
     CARTESI_ESPRESSO_READER_VERSION   = "0.2.3-node-20250128"
     CARTESI_IMAGE_KERNEL_VERSION      = "0.20.0"
+    CARTESI_LINUX_KERNEL_VERSION      = "6.5.13-ctsi-1-v0.20.0"
     CARTESI_MACHINE_EMULATOR_VERSION  = "0.18.1"
+    CARTESI_PAYMASTER_VERSION         = "0.2.0"
     CARTESI_ROLLUPS_GRAPHQL_VERSION   = "2.3.8"
+    CARTESI_ROLLUPS_NODE_BASE_IMAGE   = "docker.io/library/debian:bookworm-20250317-slim@sha256:1209d8fd77def86ceb6663deef7956481cc6c14a25e1e64daec12c0ceffcc19d"
     CARTESI_ROLLUPS_NODE_VERSION      = "2.0.0-alpha.1"
+    CARTESI_SDK_BASE_IMAGE            = "docker.io/library/node:22-bookworm-slim@sha256:bac8ff0b5302b06924a5e288fb4ceecef9c8bb0bb92515985d2efdc3a2447052"
```

Comment:
Isn't it better to use the runtime image as base?

Reply:
Comparing the size, the difference is negligible, I sent a commit that uses a single base image.

So we have what you suggested, base -> runtime -> sdk.